pkg/database/models/case.go

```diff
@@ -10,6 +10,8 @@ type Case struct {
 	DoctorID       uint
 	PatientName    string
 	DoctorName     string
```

Comment:
maybe, we can delete `DoctorID`、`Department` in case since Registeration contains them.